src/Components/Agents.md

```diff
@@ -0,0 +1,212 @@
+# Working on Issues in the Components Area
```

Comment:
That's been addressed in https://github.com/dotnet/aspnetcore/pull/64603, to be reverted.